core/src/main/scala/akka/persistence/r2dbc/internal/R2dbcSerializer.scala

```diff
@@ -40,6 +40,7 @@ import akka.serialization.SerializerWithStringManifest
   }
 
   private def offsetToBinary(offset: TimestampOffset): Array[Byte] = {
+    // Note: this is the same offset storage mechanism we use in spanner, encoding timestamp + each seen pid+seqNr when needed (same timestamp)
```

Comment:
yes, the serializer here isn't really used, unless reading (source provider) from slices and storing offset with another projection impl than r2dbc